### PR TITLE
Add link to ETCD/JDBC Registry Guideline

### LINKED ISSUE #15591: How use ETCD as externalRegistry in k8s
How use ETCD as externalRegistry in k8s?

can not find any docs

### PR DESCRIPTION
**Pull request by pegasas**
## Purpose of the pull request

Add link to ETCD/JDBC Registry Guideline
fix https://github.com/apache/dolphinscheduler/issues/15591

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: docs/docs/en/architecture/configuration.md [hunk 1]
```diff
@@ -162,9 +162,9 @@ The default configuration is as follows:
 
 Note that DolphinScheduler also supports database configuration through `bin/env/dolphinscheduler_env.sh`.
 
-### Zookeeper related configuration
+### Registry related configuration
 
-DolphinScheduler uses Zookeeper for cluster management, fault tolerance, event monitoring and other functions. Configuration file location:
+DolphinScheduler uses Zookeeper for cluster management by default, fault tolerance, event monitoring and other functions. Configuration file location:
```
Comment (rickchengx) [inline docs/docs/en/architecture/configuration.md]:
```suggestion
DolphinScheduler uses Zookeeper by default for cluster management, fault tolerance, event monitoring and other functions. Configuration file location:
```

File: docs/docs/en/architecture/configuration.md [hunk 2]
```diff
@@ -187,6 +187,9 @@ The default configuration is as follows:
 
 Note that DolphinScheduler also supports zookeeper related configuration through `bin/env/dolphinscheduler_env.sh`.
 
+For ETCD Registry, please see more details on [link](https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-registry/dolphinscheduler-registry-plugins/dolphinscheduler-registry-etcd/README.md).
```
Comment (EricGao888) [inline docs/docs/en/architecture/configuration.md:194]:
``` markdown
## Task Parameters

[//]: # (TODO: use the commented anchor below once our website template supports this syntax)
[//]: # (- Please refer to [DolphinScheduler Task Parameters Appendix]&#40;appendix.md#default-task-parameters&#41; `Default Task Parameters` section for default parameters.)

```

This reminds me of an old issue. As the comment shows above, I'm wondering whether our website has supported markdown anchor? @zhongjiajie

**Review by EricGao888 — DISMISSED** [review]

File: docs/docs/en/architecture/configuration.md [hunk 3]
```diff
@@ -162,9 +162,9 @@ The default configuration is as follows:
 
 Note that DolphinScheduler also supports database configuration through `bin/env/dolphinscheduler_env.sh`.
 
-### Zookeeper related configuration
+### Registry related configuration
```
Comment (EricGao888) [inline docs/docs/en/architecture/configuration.md]: I think we should capitalize every word of a sub-title. `Registry Related Configuration`

**Review by ruanwenjun — DISMISSED** [review]
LGTM, please resolve the conflicts.

**Review by EricGao888 — DISMISSED** [review]

**Review by rickchengx — DISMISSED** [review]
+1, @pegasas please resolve the conflicts

**Comment by pegasas** [comment]
Thanks, Done.

**Review by rickchengx — APPROVED** [review]

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Comment by ruanwenjun** [comment]
I change the milestone to 3.2.2.

**Pull request merged**